app/routes/interestgroups/components/InterestGroupMemberList.css

```diff
@@ -32,3 +40,31 @@
 .row i {
   margin-right: 5px;
 }
+
+.leader {
+  font-weight: 900;
+  text-shadow: 0.0625em 0.125em 0.875em var(--goldblur);
+  color: var(--leadergold);
+}
+
+.coleader {
+  font-weight: 750;
+  text-shadow: 0.0625em 0.125em 0.875em var(--silverblur);
+  color: var(--coleadersilver);
+}
+
+.leadericon {
+  margin: 0.2em 0.2em 0;
+  filter: drop-shadow(0.0625em 0.125em 0.625em var(--goldblur));
```

Comment:
Could we maybe reduce the shadow a tiny bit? I think it's cool on dark mode, but on light mode it feels kinda over the top. I'm all for adding shadow, but I think I'd prefer a smaller one.